### PR TITLE
Allow to pass aria-label to icons

### PR DESCRIPTION
**Pull request by gregaubert**
**Checklist**

* [ ] Functional validation
* [ ] Documentation update
* [ ] Update changelog

### CODE REVIEW

**Review by siegfried-ehret-sonarsource — APPROVED** [review]

**Review by aletsiram-castro-sonarsource — APPROVED** [review]
Yeah!!

File: components/icons/Icon.tsx [hunk 1]
```diff
@@ -21,12 +21,14 @@ import * as React from 'react';
 import { Theme, ThemeConsumer } from '../theme';
 
 export interface IconProps {
+  'aria-label'?: string;
```
Comment (wouter-admiraal-sonarsource) [inline components/icons/Icon.tsx]: As we're now basically allowing _any_ prop, I wonder if we should restrict to `aria-label` 🤔 ? Sometimes, `aria-hidden` could be interesting as well, for instance. Should we do something like:

```
export interface IconProps extends React.AriaAttributes {
```

Or even:

```
export interface IconProps extends React.HTMLAttributes<{}> {
```

?

**Review by wouter-admiraal-sonarsource — APPROVED** [review]
LGTM 👍 !

File: components/icons/Icon.tsx (hunk 1 above)
Comment (gregaubert) [reply components/icons/Icon.tsx]: Ah yeah good idea, I'll extends the `React.AriaAttributes`!

**Pull request merged**